server/routes/review.js

```diff
@@ -0,0 +1,14 @@
+const express = require("express");
+const router = express.Router();
+const { validateReview } = require("../validate");
+const protect = require("../middleware/auth");
+const {
+  createReview,
+  getReviews,
+} = require("../controllers/review");
+
+router.route("/:profileId").post(protect, validateReview, createReview);
```

Comment:
```suggestion
router.route("/").post(protect, validateReview, createReview);
```
We can send the sitter's profile id in the request body :)